src/Core/Jobs/BaseJobsHostedService.cs

```diff
@@ -68,23 +68,37 @@ public virtual async Task StartAsync(CancellationToken cancellationToken)
             {
                 foreach (var (job, trigger) in Jobs)
                 {
-                    var dupeT = await _scheduler.GetTrigger(trigger.Key);
-                    if (dupeT != null)
+                    for (var retry = 0; retry < 10; retry++)
```

Comment:
It looks like this will just proceed after failing 10 times.

We either need to try forever, or throw below this for loop if the job isn't started.